cfme/containers/provider/__init__.py

```diff
@@ -348,10 +349,9 @@ def resetter(self):
 @navigator.register(ContainersProvider, 'Edit')
 class Edit(CFMENavigateStep):
     VIEW = ContainersProviderEditView
-    prerequisite = NavigateToSibling('All')
+    prerequisite = NavigateToSibling('Details')
 
     def step(self):
-        sel.check(Quadicon(self.obj.name, self.obj.quad_name).checkbox())
         cfg_btn('Edit Selected Containers Provider')
```

Comment:
usually such items have different names on Details and All pages.
Please double check that this item name is correct